src/xrSound/stdafx.cpp

```diff
@@ -15,4 +15,6 @@
 #pragma comment(lib, "OpenAL32B.lib")
 #pragma comment(lib, "dsoundb.lib")
 //#pragma comment(lib, "xrAPI.lib")
+#elif DEBUG
+#pragma comment(lib, "xrCDB.lib") // Xottab_DUTY: For some reason it wont compile without it.
```

Comment:
Why would you put this back into the pre-compiled header when in the last PR you moved library inputs to the project files?